### DISCUSSION TITLE
I cannot finish a kata because of broken tests

### DISCUSSION THREAD

**Discussion by alvaronaschez** [codewars/codewars.com#3245] [General]
After spending some time coding a kata, I'm not passing the random tests because the tests are wrong, not my solution.

Apparently it is a known issue, but no one has fixed it in several months. 

It is frustrating and at this point I'm thinking about using Leetcode or Hackerrank instead. So I know if I finish an exercise and the solution is right I'm going to pass the tests. I wouldn't like to do so because I like codewars, but it is not the first time I'm finding this kind of issue.

The kata is [Ranking Poker Hands](https://www.codewars.com/kata/5739174624fc28e188000465) and the broken test I cannot pass includes this poker hand: `3C KH 5D 5S KH` which is invalid because it contains the king of hearts twice.

**Comment by hobovsky** [comment]
Github discussions are not meant to discuss problems with specific kata. If you encounter a problem with tests, kata setup, or any other issue related to a specific challenge, the desired flow is to raise an issue in the kata discourse, and/or report the issue in the `#fixing` channel of Codewars Discord.

Having said that, I checked the C tests of kata you mentioned, and I fixed duplicate occurrences of the KH card. Please verify if your issue is fixed.

I am going to close this discussion as off-topic for CW Github. In case of furter problems with the kata, please continue reporting them in the kata discourse, or on CW Discord.
